sdk/core/core-http/src/nodeFetchHttpClient.browser.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
do we still need this? I'm curious how the browser is pulling in `NodeFetchHttpClient`

Reply:
ohh probably not. I will confirm in a bit!